engine/src/main/resources/application.properties

```diff
@@ -180,7 +180,7 @@ quarkus.rest-client.internal-google-chat.trust-store-type=${clowder.endpoints.no
 %test.quarkus.mailer.mock=false
 
 # This will move out of the engine after the split
-notifications.camel.kafka-return-topic=platform.notifications.fromcamel
+mp.messaging.fromcamel.topic=platform.notifications.fromcamel
```

Comment:
Why this one doesn't have the `incoming` like `mp.messaging.incoming.ingress.topic` ?

Reply:
This config key is a "hack" of the config source lib. The prefix has to be `mp.messaging` and the suffix has to be `topic`, but if `incoming` or `outgoing` is present in the middle of the key, Quarkus complains because other config keys should be provided and the app doesn't start. If `incoming` and `outgoing` are missing, the config key is ignored by Quarkus and the app can start.

Reply:
That "hack" is currently used in `eventing-integrations`. It's not pretty but transforming it to something cleaner isn't my immediate priority.